source/appModules/searchui.py

```diff
@@ -25,3 +29,7 @@ def chooseNVDAObjectOverlayClasses(self,obj,clsList):
 		elif isinstance(obj, UIA):
 			if obj.UIAElement.cachedAutomationID == "SearchTextBox":
 				clsList.insert(0, StartMenuSearchField)
+			# #10329: Since 2019, some suggestion items are grouped inside another suggestions list item.
+			# Because of this, result details will not be announced like in the past.
+			elif obj.role == controlTypes.ROLE_LISTITEM and isinstance(obj.parent, SuggestionListItem):

```

Comment:
Wouldn't it be possible to add this extra check to the `findOverlayClasses` function in NVDAObjects.UIA? I have the feelling that current code scatters the places where `SuggestionListItem` is added as an overlay.